testing/mariadb-connector-odbc/APKBUILD

```diff
@@ -0,0 +1,37 @@
+# Contributor: Joe Searle <joe@jsearle.net>
+# Maintainer: Joe Searle <joe@jsearle.net>
+pkgname="mariadb-connector-odbc"
+pkgver="3.1.3"
+pkgrel=0
+pkgdesc="MariaDB Connector/ODBC is a standardized, LGPL licensed database driver using the industry standard ODBC API."
+url="https://downloads.mariadb.org/connector-odbc/"
+arch="all"
+license="LGPL-2.1-only"
+depends="unixodbc"
+makedepends="cmake unixodbc unixodbc-dev mariadb-connector-c mariadb-connector-c-dev mariadb-static"
+install=""
```

Comment:
empty variables can be removed for brevity

Reply:
empty variables removed.